server/Server.hpp

```diff
@@ -53,6 +54,8 @@ class Server
 	void waitEvents(void);
 	void addPipes(const int writeFd, const int readFd, Router* const router);
     int getRequestError(const int client_sockfd);
+    void addCookie(const std::string& key, const std::string& value);
+    const std::string& getCookie(const std::string& key);
```

Comment:
```suggestion
    const std::string& getCookie(const std::string& key) **const**;
```
멤버변수의 변조가 없으므로 const를 붙이는 것이 좋아보입니다.

Reply:
감사합니당!